### ISSUE TITLE
All declarations of 'Matchers' must have identical type parameters.

### ISSUE THREAD

**Issue by blackraindrop** [testing-library/jest-dom#152]
### Relevant code or config:

```js
@testing-library/jest-dom` version:4.2.0
@types/jest:24.0.20

interface Matchers<R, T> {}
```

**Comment by gnapse** [comment]
Is this closed by #150?

**Comment by gnapse** [comment]
> Is this closed by #150?

I'm assuming the answer to the above question is "Yes", so I'm closing this.
